### PR TITLE
styled(std): remove unnecessary if else block

### PR DESCRIPTION
**Pull request by behnammodi**
In the title is explained :)

### CODE REVIEW

File: std/fs/ensure_symlink.ts [hunk 1]
```diff
@@ -28,13 +28,13 @@ export async function ensureSymlink(src: string, dest: string): Promise<void> {
 
   await ensureDir(path.dirname(dest));
 
-  if (Deno.build.os === "windows") {
-    await Deno.symlink(src, dest, {
+  const options = Deno.build.os === "windows"
```
Comment (kitsonk) [inline std/fs/ensure_symlink.ts:31]:
```suggestion
  const options: Deno.SymlinkOptions = Deno.build.os === "windows"
```

File: std/fs/ensure_symlink.ts [hunk 2]
```diff
@@ -60,11 +60,12 @@ export function ensureSymlinkSync(src: string, dest: string): void {
   }
 
   ensureDirSync(path.dirname(dest));
-  if (Deno.build.os === "windows") {
-    Deno.symlinkSync(src, dest, {
+
+  const options = Deno.build.os === "windows"
```
Comment (kitsonk) [inline std/fs/ensure_symlink.ts:64]:
```suggestion
  const options: Deno.SymlinkOptions = Deno.build.os === "windows"
```

**Review by kitsonk — CHANGES_REQUESTED** [review]
You need to make changes in order to have CI pass.

**Comment by behnammodi** [comment]
@kitsonk Thank you, I did it, I'm very enjoying this project, It's awesome :)

**Review by kitsonk — APPROVED** [review]
LGTM

**Comment by nurmohammed840** [comment]
I have a question in code snippet, please read the comments below... 
```ts
export async function ensureSymlink(src: string, dest: string): Promise<void> {
	const srcStatInfo = await Deno.lstat(src);  // 1. what is the point to `wait` here!?
	const srcFilePathType = getFileInfoType(srcStatInfo);  // I mean `srcFilePathType` would or could remain unused!

	if (await exists(dest)) {   // because  if this code block run then `ensureSymlink` will ether  throw 'err' or return 'undefiend'
		const destStatInfo = await Deno.lstat(dest);
		const destFilePathType = getFileInfoType(destStatInfo);
		if (destFilePathType !== "symlink") {
			throw new Error(`Ensure path exists, expected 'symlink', got '${destFilePathType}'`);
		}
		return;  
	}

	await ensureDir(path.dirname(dest)); 

+     const srcStatInfo = await Deno.lstat(src);  // those code line should belong here. 
+     const srcFilePathType = getFileInfoType(srcStatInfo); 

	const options: Deno.SymlinkOptions | undefined = isWindows
		? { type: srcFilePathType === "dir" ? "dir" : "file" } // even though  `srcFilePathType` read here only!
		: undefined;

	await Deno.symlink(src, dest, options);
}
```
Am I right ?

**Comment by behnammodi** [comment]
@nurmohammed840 I fixed thank you 👏🏻

**Pull request merged**